net.jammr.jammr.appdata.xml

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+    <id>net.jammr.jammr</id>
+    <metadata_license>MIT</metadata_license>
+    <name>jammr client</name>
+    <summary>Play with musicians over the internet</summary>
+    <description>
+        <p>
+            jammr lets you jam together online. It is possible to play jam
+            sessions with musicians around the world over the internet. Any
+            instrument or microphone that can be recorded on your computer can
+            be used to play.
+        </p>
+    </description>
+    <categories>
+        <category>Audio</category>
+        <category>AudioVideo</category>
+        <category>Music</category>
+    </categories>
+    <url type="homepage">https://jammr.net/</url>
+    <url type="faq">https://jammr.net/faq.html</url>
+    <url type="help">https://forum.jammr.net/1/</url>
+    <url type="contact">https://jammr.net/contact.html</url>
+    <launchable type="desktop-id">net.jammr.jammr.desktop</launchable>
+    <project_license>GPL-2.0-or-later</project_license>
+    <developer_name>jammr ltd</developer_name>
+    <releases>
```

Comment:
You need OARS tag https://hughsie.github.io/oars/

Reply:
Thanks, will add in the next revision.